### DISCUSSION TITLE
Include Flow Part in lucene index?

### DISCUSSION THREAD

**Discussion by devlife** [Lombiq/Helpful-Extensions#124] [Q&A]
is it possible to include Flow Part in a lucene index?

**Comment by Piedone** [comment]
Did you rather mean this for the [Orchard Core discussions](https://github.com/OrchardCMS/OrchardCore/discussions)?

**Reply by devlife** [reply]
I put it here because I'm trying to index the Flow Part of a Page but I can move it if that's a more appropriate place.

**Reply by Piedone** [reply]
Yes, please, since this is not only related to Helpful Extensions.

**Comment by devlife** [comment]
Moved to https://github.com/OrchardCMS/OrchardCore/discussions/13477

**Reply by Piedone** [reply]
Thanks!

**Reply by devlife** [reply]
In case you were interested, the FlowPartIndexHandler is set to be released with 1.6

**Reply by Piedone** [reply]
Great, thanks for coming back here.
